src/services/opensrp/tests/index.test.ts

```diff
@@ -26,20 +26,26 @@ describe('services/OpenSRP', () => {
     });
   });
 
+  it('getPayloadOptions works', async () => {
+    const output = {
+      headers: {
+        accept: 'application/json',
+        authorization: 'Bearer hunter2',
+        'content-type': 'application/json;charset=UTF-8',
+      },
+      method: 'POST',
+    };
+    const signal = new AbortController().signal;
+    expect(getPayloadOptions(signal, 'POST')).toEqual({ ...output });
```

Comment:
@ciremusyoka what happens to the report if you add another `expect()` here? (because before there were two?)

Reply:
> I think either we don't yet quite understand how coveralls interprets the jest-generated coverage reports or there is something wrong with the actual reports

Agreed, we should do some research here to see how it's interpreted. Can you have over 100% coverage of a file?

Reply:
@cKellyDesign one of the assertions was for the `getPayload` method which was being used before we started using `openSrpService` class. With the current implementation the method is no longer used so i removed it plus it's tests

Reply:
> Can you have over 100% coverage of a file?

:rofl: it would break some mathematical and physical laws. I however do not think that a drop coverage should have risen from the changes in this pr.